lib/tasks/import_coastalstats.rake

```diff
@@ -0,0 +1,47 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task :coastalstats, [:csv_file] => [:environment] do
+    filename = 'coastline_coverage'
+    csv_file = "#{Rails.root}/lib/data/#{filename}/#{filename}.csv"
+
+    import_csv_file(csv_file)
+  end
+
+  def import_csv_file(csv_file)
+    CSV.foreach(csv_file, headers: true, encoding: 'utf-8') do |row|
+      # temporary workaround to catch only one buffer
+      next unless row['buffer'] == '30'
+      habitats = Habitat.all
+      name = row['iso3'] || row['name']
+      geo_entity = fetch_geo_entity(name)
```

Comment:
I'm getting an error when importing which is probably related to this method conflicting with the one in the prebakedstats import.
If you need the implementation of this to be different I would suggest to rename it